icon-registry/icons/status-failed-outline_x16.svg

```diff
@@ -1,3 +1,4 @@
-<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.29289 10.7071ZM10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L10.7071 6.70711ZM5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L5.29289 9.29289ZM13 8C13 10.7614 10.7614 13 8 13V15C11.866 15 15 11.866 15 8H13ZM8 13C5.23858 13 3 10.7614 3 8H1C1 11.866 4.13401 15 8 15V13ZM3 8C3 5.23858 5.23858 3 8 3V1C4.13401 1 1 4.13401 1 8H3ZM8 3C10.7614 3 13 5.23858 13 8H15C15 4.13401 11.866 1 8 1V3ZM5.29289 6.70711L7.29289 8.70711L8.70711 7.29289L6.70711 5.29289L5.29289 6.70711ZM7.29289 8.70711L9.29289 10.7071L10.7071 9.29289L8.70711 7.29289L7.29289 8.70711ZM9.29289 5.29289L7.29289 7.29289L8.70711 8.70711L10.7071 6.70711L9.29289 5.29289ZM7.29289 7.29289L5.29289 9.29289L6.70711 10.7071L8.70711 8.70711L7.29289 7.29289Z" fill="#E45770"/>
+<svg viewBox="0 0 16 16" fill="none"
+  xmlns="http://www.w3.org/2000/svg">
+  <path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.29289 10.7071ZM10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L10.7071 6.70711ZM5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L5.29289 9.29289ZM13 8C13 10.7614 10.7614 13 8 13V15C11.866 15 15 11.866 15 8H13ZM8 13C5.23858 13 3 10.7614 3 8H1C1 11.866 4.13401 15 8 15V13ZM3 8C3 5.23858 5.23858 3 8 3V1C4.13401 1 1 4.13401 1 8H3ZM8 3C10.7614 3 13 5.23858 13 8H15C15 4.13401 11.866 1 8 1V3ZM5.29289 6.70711L7.29289 8.70711L8.70711 7.29289L6.70711 5.29289L5.29289 6.70711ZM7.29289 8.70711L9.29289 10.7071L10.7071 9.29289L8.70711 7.29289L7.29289 8.70711ZM9.29289 5.29289L7.29289 7.29289L8.70711 8.70711L10.7071 6.70711L9.29289 5.29289ZM7.29289 7.29289L5.29289 9.29289L6.70711 10.7071L8.70711 8.70711L7.29289 7.29289Z" fill="#E45770" class="icon-dark"/>
```

Comment:
did you not want these failed ones to use currentColor?

Reply:
I might have missed that

Reply:
```suggestion
  <path d="M6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM9.29289 10.7071C9.68342 11.0976 10.3166 11.0976 10.7071 10.7071C11.0976 10.3166 11.0976 9.68342 10.7071 9.29289L9.29289 10.7071ZM10.7071 6.70711C11.0976 6.31658 11.0976 5.68342 10.7071 5.29289C10.3166 4.90237 9.68342 4.90237 9.29289 5.29289L10.7071 6.70711ZM5.29289 9.29289C4.90237 9.68342 4.90237 10.3166 5.29289 10.7071C5.68342 11.0976 6.31658 11.0976 6.70711 10.7071L5.29289 9.29289ZM13 8C13 10.7614 10.7614 13 8 13V15C11.866 15 15 11.866 15 8H13ZM8 13C5.23858 13 3 10.7614 3 8H1C1 11.866 4.13401 15 8 15V13ZM3 8C3 5.23858 5.23858 3 8 3V1C4.13401 1 1 4.13401 1 8H3ZM8 3C10.7614 3 13 5.23858 13 8H15C15 4.13401 11.866 1 8 1V3ZM5.29289 6.70711L7.29289 8.70711L8.70711 7.29289L6.70711 5.29289L5.29289 6.70711ZM7.29289 8.70711L9.29289 10.7071L10.7071 9.29289L8.70711 7.29289L7.29289 8.70711ZM9.29289 5.29289L7.29289 7.29289L8.70711 8.70711L10.7071 6.70711L9.29289 5.29289ZM7.29289 7.29289L5.29289 9.29289L6.70711 10.7071L8.70711 8.70711L7.29289 7.29289Z" fill="currentColor" class="icon-dark"/>
```